f2/src/forms/ContactInfoForm.js

```diff
@@ -22,11 +21,15 @@ import { formDefaults } from './defaultValues'
 export const validate = (values) => {
   const errors = {}
   //condition for an error to occur: append a lingui id to the list of error
-  if (values.email !== '' && addrs(values.email) == null) {
+  // reference https://www.w3resource.com/javascript/form/email-validation.php
+  // if (!/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(values.email)) {
+  if (!/^\w+([-]?\w+)*@\w+([-]?\w+)*(\.\w{2,3})+$/.test(values.email)) {
```

Comment:
suggesting this regex: `^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]{2,})+$` [source](https://emailregex.com/)